### DISCUSSION TITLE
Event post type

### DISCUSSION THREAD

**Discussion by huumn** [stackernews/stacker.news#3011] [Ideas]
Basically it's like a discussion post but allows you to specify time/date/location. If a sub supports event post types we could display a calendar view for the events in the sub ...

In this way this would allow us to create communal calendars.

**Comment by dillon-co** [comment]
Should I make location a required field? I was imagining no in because of things like the halving and stuff. 


also, wondering if ticket 633 should include crossposting to Nostr?

**Comment by huumn** [comment]
> Should I make location a required field? I was imagining no in because of things like the halving and stuff.

I vote for no.

> also, wondering if ticket 633 should include crossposting to Nostr?

I think it should be an independent issue.
